src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -375,7 +374,7 @@ namespace winrt::TerminalApp::implementation
     // - <none>
     void TerminalPage::HandoffToElevated(const CascadiaSettings& settings)
     {
-        if (!_startupActions)
+        if (_startupActions.empty())
```

Comment:
Pretty sure this was a bug. Handing off to elevated with 0 tabs doesn't seem useful to me.